packages/storefront-events-collector/src/schemas.ts

```diff
@@ -12,7 +12,7 @@ const schemas = {
     SEARCH_INPUT_SCHEMA_URL: "iglu:com.adobe.magento.entity/search-input/jsonschema/1-0-11",
     SEARCH_RESULT_CATEGORY_SCHEMA_URL: "iglu:com.adobe.magento.entity/search-result-category/jsonschema/1-0-1",
     SEARCH_RESULT_PRODUCT_SCHEMA_URL: "iglu:com.adobe.magento.entity/search-result-product/jsonschema/1-0-2",
-    SEARCH_RESULTS_SCHEMA_URL: "iglu:com.adobe.magento.entity/search-results/jsonschema/1-0-9",
+    SEARCH_RESULTS_SCHEMA_URL: "iglu:com.adobe.magento.entity/search-results/jsonschema/1-0-11",
```

Comment:
is this published to prod in snowplow?

Reply:
hi @grahamcrackers! yes it is!! thank you so much for remembering and for checking! i merged it to prod as part of [DINT-902](https://jira.corp.adobe.com/browse/DINT-902) - the new schema version is viewable [here](https://console.snowplowanalytics.com/f0aa80a7-0012-46d3-ad4e-5fbe36c22504/data-structures?hidden=false&rows=10&search=&sort=lastUpdate&sortO=desc). attaching screenshot too for completeness

<img width="1339" alt="Screenshot 2023-04-25 at 11 31 50" src="https://user-images.githubusercontent.com/847621/234343567-3a41bf0b-6c00-4f6e-82e8-30e7bfda1641.png">